### PR TITLE
New template for elasticsearch 

### PR DESCRIPTION
**Pull request by adhurwit**
new template for a 3-node CentOS elasticsearch

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@adhurwit__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (adhurwit). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

**Comment by adhurwit** [comment]
This should be ready to merge now.

File: elasticsearch-centos-3node/azuredeploy.json [hunk 1]
```diff
@@ -0,0 +1,381 @@
...
+    ],
+    "jumpboxScripts": [
+        "[concat(variables('scriptBaseUrl'), variables('jumpboxScript'))]",
+        "[concat(variables('scriptBaseUrl'), 'get-started.ps1')]"
+    ]
+    
+  },
+  "resources": [
+    {
+      "apiVersion": "2015-05-01-preview",
```
Comment (singhkay) [inline elasticsearch-centos-3node/azuredeploy.json]: I suggest making the api version for all resources a variable and using the GA versions 2015-06-15

**Comment by singhkay** [comment]
@template-deploy-bot test this please

File: elasticsearch-centos-3node/azuredeploy.json (hunk 1 above)
Comment (adhurwit) [inline elasticsearch-centos-3node/azuredeploy.json]: OK, making the change...

**Comment by singhkay** [comment]
@template-deploy-bot test this please

**Comment by singhkay** [comment]
@adhurwit I see what happened :) 

@template-deploy-bot test this please

**Comment by singhkay** [comment]
@adhurwit Looks good, can you change the locations back to this repo for merging

**Comment by adhurwit** [comment]
@singhkay  Thanks, done

**Comment by singhkay** [comment]
merged, thanks!

**Pull request merged**